meta-tedge-mender/conf/layer.conf

```diff
@@ -13,3 +13,5 @@ LAYERDEPENDS_meta-tedge-mender = "meta-tedge-common"
 LICENSE_PATH += "${LAYERDIR}/"
 
 LAYERSERIES_COMPAT_meta-tedge-mender = "scarthgap"
+
+MENDER_FEATURES_ENABLE:append = " mender-uboot"
```

Comment:
I'm actually surprised that you're allowed to add this to the layer.conf.

Though a better place would be to add it to the mender config snippet, `kas/config/mender-raspberrypi.yaml`, as we don't want to force people to using u-boot, so it should be on the project level to decide what bootloader they want.